src/host/main.cpp

```diff
@@ -320,6 +325,14 @@ int main(int argc, char** argv)
                                   public_rpc_address.hostname,
                                   node_address.port,
                                   rpc_address.port};
+  if (consensus == "raft")
+  {
+    consensus_type = ConsensusType::Raft;
```

Comment:
CLI11 1.8 has nicer support for enums (see https://github.com/CLIUtils/CLI11/pull/233), this is separate from this PR, but it would be nice to upgrade and use that support (or at least to ticket it for now :)).

Reply:
https://github.com/microsoft/CCF/issues/569